packages/poppler.rb

```diff
@@ -3,25 +3,25 @@
 class Poppler < Package
```

Comment:
`class Poppler < CMake`?

Reply:
No because of the `cmake_option` `-DENABLE_UNSTABLE_API_ABI_HEADERS=ON \`